subsys/bluetooth/controller/hci/hci.c

```diff
@@ -6036,18 +6036,18 @@ static void le_per_adv_sync_established(struct pdu_data *pdu_data,
 
 	se = (void *)pdu_data;
 	sep->status = se->status;
-	sep->handle = sys_cpu_to_le16(node_rx->hdr.handle);
 
-	if (sep->status != BT_HCI_ERR_SUCCESS && sep->status != BT_HCI_ERR_UNSUPP_REMOTE_FEATURE) {
+	if (se->status == BT_HCI_ERR_OP_CANCELLED_BY_HOST) {
 		return;
 	}
 
-	scan = node_rx->hdr.rx_ftr.param;
+	sep->handle = sys_cpu_to_le16(node_rx->hdr.handle);
 
-	sep->sid = scan->per_scan.sid;
 	/* Resolved address, if private, has been populated in ULL */
+	scan = node_rx->hdr.rx_ftr.param;
```

Comment:
The comment `/* Resolved address, if private, has been populated in ULL */` seems to not belong to the line.

Reply:
Updated.